addons/mail/static/src/components/message/message.xml

```diff
@@ -177,7 +177,7 @@
                             </ul>
                         </t>
                     </div>
-                    <t t-if="message.subject and threadView and threadView.thread and (threadView.thread.mass_mailing or [env.messaging.inbox, env.messaging.history].includes(threadView.thread))">
+                    <t t-if="message.subject and !message.isSubjectEqualThreadName and threadView and threadView.thread and (threadView.thread.mass_mailing or [env.messaging.inbox, env.messaging.history].includes(threadView.thread))">
```

Comment:
No change here, but note to self, because I keep double checking: `isSubjectEqualThreadName` does not have to be added to `useStore` in this case because `__state` is used in message.